src/index.rs

```diff
@@ -94,6 +95,37 @@ impl IndexWriter {
     fn commit_opstamp(&self) -> u64 {
         self.inner_index_writer.commit_opstamp()
     }
+
+    /// Delete all documents containing a given term.
```

Comment:
This needs documentation for the arguments and the return value. We also may want to figure out a better name for the method since we don't pass a term but a value.